atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -191,6 +195,15 @@ def run(self):
             "plugins-metadata": json.dumps(self.get_plugin_metadata())
         }
 
+        help_result = self.workflow.prebuild_results.get(AddHelpPlugin.key)
+        if isinstance(help_result, dict) and 'help_file' in help_result and 'status' in help_result:
+            if help_result['status'] == AddHelpPlugin.NO_HELP_FILE_FOUND:
+                annotations['help_file'] = json.dumps(None)
+            elif help_result['status'] == AddHelpPlugin.HELP_GENERATED:
+                annotations['help_file'] = json.dumps(help_result['help_file'])
+            else:
+                self.log.error("Unknown result from add_help plugin: %s", help_result)
```

Comment:
Does this get test coverage? coveralls thinks it doesn't:
https://coveralls.io/builds/12383762/source?filename=atomic_reactor%2Fplugins%2Fexit_store_metadata_in_osv3.py#L200